src/main/java/com/mongodb/jdbc/MongoSQLConnection.java

```diff
@@ -11,69 +12,31 @@
 
 public class MongoSQLConnection extends MongoConnection implements Connection {
 
-    public MongoSQLConnection(ConnectionString cs, String database, String conversionMode) {
-        super(cs, database, conversionMode);
+    public MongoSQLConnection(ConnectionString cs, String database) {
+        super(cs, database);
     }
```

Comment:
The `conversionMode` is only relevant to `MySQL`, so we should not have it in the base class.